src/test/java/hw3/ex1/Ex1Assertions.java

```diff
@@ -0,0 +1,102 @@
+package hw3.ex1;
+
+import org.openqa.selenium.WebElement;
+import org.testng.asserts.SoftAssert;
+import java.util.Arrays;
+import java.util.List;
+
+import composite.HeaderSectionComposite;
+import voids.AbstractPage;
+import voids.EpamUserIndexPage;
+
+import static org.testng.Assert.assertEquals;
+
+public class Ex1Assertions {
```

Comment:
Why do you need this class? Why do you decide do not using direct methods of the page object in the test?